package.json

```diff
@@ -27,13 +27,14 @@
     "@mdx-js/react": "^1.6.21",
     "@types/react": "^17.0.11",
     "@uniswap/analytics": "1.1.0",
-    "@uniswap/analytics-events": "1.0.5",
+    "@uniswap/analytics-events": "1.1.1",
     "algoliasearch": "^4.9.1",
     "clsx": "^1.1.1",
     "hast-util-is-element": "1.1.0",
     "intl-locales-supported": "^1.8.12",
     "react": "18",
     "react-dom": "18",
+    "react-feather": "^2.0.10",
```

Comment:
Needed for the *neutral* icon, which is not included in the already installed *heroicons* package. In fact, they don't have any icons for a neutral CTA